test/extended/testdata/service_catalog/ups-deployment.yaml

```diff
@@ -4,7 +4,7 @@ kind: Template
 metadata:
   name: ups-deployment-template
 objects:
-- apiVersion: extensions/v1beta1
+- apiVersion: apps/v1
```

Comment:
/hold
doesn't this need updating the bindata? and why isn't verify screaming?

Reply:
it does, filling a BZ

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1784324, thanks.